spec/index.bs

```diff
@@ -4799,140 +4800,145 @@ enum GPUStoreOp {
-
-    <div class=validusage dfn-for=GPURenderPassEncoder.setViewport>
-        <dfn abstract-op>setViewport Valid Usage</dfn>
-
-        1. |width| must be greater than `0`.
-        1. |height| must be greater than `0`.
-        1. |minDepth| must be greater than or equal to `0.0` and less than or equal to `1.0`.
-        1. |maxDepth| must be greater than or equal to `0.0` and less than or equal to `1.0`.
-
-    </div>
-
-    Issue: Allowed for GPUs to use fixed point or rounded viewport coordinates
-
-</div>
-
-### <dfn method for=GPURenderPassEncoder>setScissorRect(x, y, width, height)</dfn> ### {#GPURenderPassEncoder-setScissorRect}
-
-<div algorithm="GPURenderPassEncoder.setScissorRect">
-    <strong>|this|:</strong> of type {{GPURenderPassEncoder}}.
-
-    **Arguments:**
-        - {{GPUIntegerCoordinate}} |x| - minimum X value of the scissor rectangle in pixels
-        - {{GPUIntegerCoordinate}} |y| - minimum Y value of the scissor rectangle in pixels
-        - {{GPUIntegerCoordinate}} |width| of the scissor rectangle in pixels
-        - {{GPUIntegerCoordinate}} |height| of the scissor rectangle in pixels
-
-    **Returns:** void
+<dl dfn-type="method" dfn-for="GPURenderPassEncoder">
+    : <dfn>setViewport(x, y, width, height, minDepth, maxDepth)</dfn>
+    ::
 
-    Sets the scissor rectangle used during the rasterization stage.
-    After transformation into viewport coordinates any fragments which fall outside the scissor
-    rectangle will be discarded.
+        Sets the viewport used during the rasterization stage to linearly map from normalized device
+        coordinates to viewport coordinates.
 
-    On the [=Device timeline=], the following steps occur:
+        <div algorithm="GPURenderPassEncoder.setViewport">
+            **Called on:** {{GPURenderPassEncoder}} |this|.
 
-        - If the [$GPURenderPassEncoder.setScissorRect/setScissorRect Valid Usage$] rules are met:
+            **Arguments:**
+            <ul dfn-type=argument dfn-for="GPURenderPassEncoder/setViewport(x, y, width, height, minDepth, maxDepth)">
+                - |x|: `float` <dfn>x</dfn> minimum X value of the viewport in pixels
+                - |y|: `float` <dfn>y</dfn> minimum Y value of the viewport in pixels
+                - |width|: `float` <dfn>width</dfn> of the viewport in pixels
+                - |height|: `float` <dfn>height</dfn> of the viewport in pixels
+                - |minDepth|: `float` <dfn>minDepth</dfn> Minimum depth value of the viewport
+                - |maxDepth|: `float` <dfn>maxDepth</dfn> Maximum depth value of the viewport.
+            </ul>
 
-            - Set the scissor rectangle to the extents |x|, |y|, |width|, and |height|.
+            **Returns:** void
 
-    <div class=validusage dfn-for=GPURenderPassEncoder.setScissorRect>
-        <dfn abstract-op>setScissorRect Valid Usage</dfn>
+            - Issue the following steps on the [=Device timeline=] of |this|:
```

Comment:
![image](https://user-images.githubusercontent.com/606355/88866405-7891c800-d1bf-11ea-9431-a0da9c295fb0.png)
(and the rest)